lib/index.js

```diff
@@ -19,6 +20,12 @@ var KISSmetrics = module.exports = integration('KISSmetrics')
   .retries(2);
 
 /**
+ * Use a POST request if the GET query string payload exceeds 2KB.
+ */
+
+var MAX_GET_PAYLOAD = 1 << 11;
```

Comment:
tbh literally sounds a simpler for future engs reading the code so I'd go with that :)